packages/cashier/src/Components/Form/crypto-withdraw-form.jsx

```diff
@@ -135,9 +138,12 @@ const CryptoWithdrawForm = ({
                             </div>
                             <CryptoFiatConverter
                                 from_currency={crypto_currency}
+                                resetConverter={resetConverter}
                                 to_currency={current_fiat_currency || DEFAULT_FIAT_CURRENCY}
                                 validateFromAmount={validateWithdrawFromAmount}
                                 validateToAmount={validateWithdrawToAmount}
+                                onChangeConverterFromAmount={onChangeConverterFromAmount}
```

Comment:
@artsemkavaleu-deriv There is an issue with your refactor. You need to also clear passing these props from crypto-fiat-converter component, and pass them from the higher level component everywhere this component is used.